lib/main/scrapper.js

```diff
@@ -0,0 +1,23 @@
+var cheerio = require('cheerio');
+var http = require('http');
+var url = require('url');
+
+function Scrapper(uri, success_callback, error_callback){
+  if(uri.indexOf('http') < 0){
+    uri = "http://" + uri;
+  }
+  http.get(uri, function(response){
+    var body = '';
```

Comment:
Check how many times this callback method is called. I believe that you get complete response in the callback and there is no need to assemble it chunk by chunk. Check difference between http.get and http.request

Rest is awesome and thumbs up for error handling. :)